src/main/java/com/codeborne/selenide/impl/IO.java

```diff
@@ -0,0 +1,62 @@
+package com.codeborne.selenide.impl;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class IO {
```

Comment:
Not sure about `IO` class name, maybe better to call it for example `FileUtililities` because now it looks like simple utility class.
Also `impl` folder required some interface implementation.